annotation-processor/src/main/java/org/hibernate/validator/ap/util/MessagerAdapter.java

```diff
@@ -114,7 +114,7 @@ private void report(ConstraintCheckIssue issue, Kind kind) {
 		String message = errorMessages.getString( issue.getMessageKey() );
 
 		if ( issue.getMessageParameters() != null ) {
-			message = MessageFormat.format( message, issue.getMessageParameters() );
+			message = new MessageFormat( message, Locale.ENGLISH ).format( issue.getMessageParameters() );
```

Comment:
I though about it ... the reason why I left it as is is because the messages here use `{0}` such things for inserting parameters. And it will require to update messages to use something like `%s` or `%1$s`. And I wasn't sure if that was a _safe_ change to make... if it is, I'd say - let's change it to use String.format